src/components/Note.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import DeleteIcon from "@material-ui/icons/Delete";
+
+function Note(props) {
+  return (
+    <div className="note">
+      <h1>{props.title}</h1>
+      <p>{props.content}</p>
+      <button
+        onClick={() => {
+          props.onDelete(props.id);
```

Comment:
I find this hard to read. Would it be possible to have something shorter, something that fits in one line? Like
`<button onClick={props.onDelete(props.id)}>`?